# This message holds data from pressure sensors in robot's single foot
# (FSR, Force Sensing Resistor). It is used to measure ZMP
# position.
#
# tx - Axis x moment
# ty - Axis y moment
# fz - Axis z force
# uch0 - uch3 - raw data from FSR

float32 tx
float32 ty
float32 fz
float32 uch0
float32 uch1
float32 uch2
float32 uch3

